app/src/processing/app/Editor.java

```diff
@@ -1374,6 +1374,24 @@ public void actionPerformed(ActionEvent e) {
 
     menu.addSeparator();
 
+    JMenuItem increaseFontSizeItem = newJMenuItem(tr("Increase Font Size"), '=');
```

Comment:
Ah! I hadn't thought about other keyboards.
One convention I'm familiar with is to use - and = to avoid having to use a shift key to increase but not decrease the font size. (Plus can only be used with a shift modifier on a US keyboard.)

Perhaps `ctrl + '+'` should be the primary shortcut and `ctrl + '='` be secondary?